code/src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react' 
+
+const Button = ({buttonText, onPageChange, buttonType="button"}) => {
+    return (
+        <button class="button" onClick={onPageChange} type={buttonType}>
+            {buttonText}
+        </button>
+    )
+}
+
+export default Button
```

Comment:
Good job adding a seperate component for a button! It will be easy to reuse the same code again!